x11-misc/lightdm/lightdm-1.32.0-r3.ebuild

```diff
@@ -0,0 +1,173 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools pam qmake-utils readme.gentoo-r1 systemd vala xdg-utils
+
+DESCRIPTION="A lightweight display manager"
+HOMEPAGE="https://github.com/canonical/lightdm"
+SRC_URI="https://github.com/canonical/lightdm/releases/download/${PV}/${P}.tar.xz
+	mirror://gentoo/introspection-20110205.m4.tar.bz2"
+
+LICENSE="GPL-3 LGPL-3"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+IUSE="audit elogind +gnome +gtk +introspection non-root qt5 systemd vala X"
+
+REQUIRED_USE="^^ ( elogind systemd )
+	vala? ( introspection )"
+
+RESTRICT="test"
+
+COMMON_DEPEND="
+	>=dev-libs/glib-2.44.0:2
+	dev-libs/libgcrypt:=
+	dev-libs/libxml2
+	sys-libs/pam
+	x11-libs/libX11
+	x11-libs/libxcb:=
+	x11-libs/libXdmcp
+	x11-libs/libxklavier:=
+	audit? ( sys-process/audit )
+	gnome? ( sys-apps/accountsservice )
+	introspection? ( >=dev-libs/gobject-introspection-1 )
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtdbus:5
+		dev-qt/qtgui:5
+	)
+"
+RDEPEND="${COMMON_DEPEND}
+	acct-group/lightdm
+	acct-group/video
+	acct-user/lightdm
+	>=sys-auth/pambase-20101024-r2
+	elogind? ( sys-auth/elogind[pam] )
+	systemd? ( sys-apps/systemd[pam] )
+	X? ( x11-base/xorg-drivers )"
+
+DEPEND="${COMMON_DEPEND}
+	gnome? ( gnome-base/gnome-common )
+	X? ( x11-base/xorg-drivers )
```

Comment:
Why is `X? ( x11-base/xorg-drivers )` not in COMMON_DEPEND? Also it seems the commit message is now off.